katz/src/main/kotlin/katz/typeclasses/Applicative.kt

```diff
@@ -30,6 +32,10 @@ data class Tuple8<out A, out B, out C, out D, out E, out F, out G, out H>(val a:
 data class Tuple9<out A, out B, out C, out D, out E, out F, out G, out H, out I>(val a: A, val b: B, val c: C, val d: D, val e: E, val f: F, val g: G, val h: H, val i: I)
 data class Tuple10<out A, out B, out C, out D, out E, out F, out G, out H, out I, out J>(val a: A, val b: B, val c: C, val d: D, val e: E, val f: F, val g: G, val h: H, val i: I, val j: J)
 
+infix fun <A, B, C, D> Tuple3<A, B, C>.toT(d: D): Tuple4<A, B, C, D> = Tuple4(this.a, this.b, this.c, d)
```

Comment:
Should this be `plus`? Also wondering what a `Semigroup` instance for each of the tuples looks like ;)